test/CheckoutSdk.Tests/TokensTests.cs

```diff
@@ -1,22 +1,55 @@
-using System;
-using System.Net;
-using Checkout.Tokens;
+using Checkout.Sdk.Common;
+using Checkout.Sdk.Tests.Mocks;
+using Checkout.Sdk.Tokens;
 using Shouldly;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
 
-namespace Checkout.Tests
+namespace Checkout.Sdk.Tests
 {
-    class describe_tokens : ApiTest
+    public class TokensTests : IClassFixture<ApiTestFixture>
```

Comment:
Order of methods, public > private

Reply:
I try to write code that is readable from top to bottom, so if you read a method and see inside that another one is called (in this case 'CreateValidRequest()' within 'ValidCardRequest()') it is placed just below. This way new concepts can introduced incrementally and it's harder to get lost.